web/modules/custom/dpl_search/src/Controller/DplSearchResultController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\dpl_search\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * This is used for demoing the react components.
+ */
+class DplSearchResultController extends ControllerBase {
+
+  /**
+   * Demo react rendering.
+   *
+   * @return mixed[]
+   *   Render array.
+   */
+  public function index(): array {
+
```

Comment:
Please remove the blank place. Nitpicky, I know.

Reply:
👍